Answers.md

```diff
@@ -0,0 +1,11 @@
+
+
+1. The HTML semantics are rules of setting up HTML content so that web page layout will be structured in smooth way. Each HTML content is contained within an HTML tag that have different characteristics that follow different rules. HTML Semanetics are like a skeletal structure for a human being. If the each bone is not placed in its correct place, the human body will have difficulty performing basic physical functions. 
```

Comment:
There is a difference between semantic HTML and presentational HTML. For instance, what is the difference between `<section>` and `<div>`?